src/ddmd/astnull.d

```diff
@@ -0,0 +1,1814 @@
+module ddmd.astnull;
```

Comment:
Probably not. It was there in the initial version and now seems to have been removed but was kept by git rebase. I'll remove it.